Watchman.DomainModel/Settings/GetConfigurationQueryHandler.cs

```diff
@@ -15,7 +17,22 @@ public GetConfigurationQueryHandler(ISessionFactory sessionFactory)
         public GetConfigurationQueryResult Handle(GetConfigurationQuery query)
         {
             using var session = this._sessionFactory.Create();
-            var configurationItems = session.Get<ConfigurationItem>();
+            var allConfigurationItems = session.Get<ConfigurationItem>();
+            var configurationItems = allConfigurationItems.Where(x => x.ServerId == query.ServerId);
+            var defaultConfigurationItems = allConfigurationItems.Where(x => x.ServerId == 0);
+            if (!configurationItems.Any())
+            {
+                return new GetConfigurationQueryResult(defaultConfigurationItems);
+            }
+            foreach (var configurationItem in defaultConfigurationItems)
+            {
+                if (configurationItems.FirstOrDefault(x => x.Name == configurationItem.Name) != null)
```

Comment:
```suggestion
                if (configurationItems.Any(x => x.Name == configurationItem.Name))
```
proponuję zmienić nazwę configurationItems na serverConfigurationItems

Reply:
done :)